pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1013,18 +1017,20 @@ func doActivate(ctx *domainContext, config types.DomainConfig,
 	// Do we need to copy any rw files? Preserve ones are copied upon
 	// creation
 	for _, ds := range status.DiskStatusList {
-		if ds.ReadOnly || ds.Preserve {
-			continue
-		}
-		log.Infof("Copy from %s to %s\n", ds.FileLocation, ds.ActiveFileLocation)
-		if _, err := os.Stat(ds.ActiveFileLocation); err == nil && ds.Preserve {
-			log.Infof("Preserve and target exists - skip copy\n")
-		} else if err := cp(ds.ActiveFileLocation, ds.FileLocation); err != nil {
-			log.Errorf("Copy failed from %s to %s: %s\n",
-				ds.FileLocation, ds.ActiveFileLocation, err)
-			status.LastErr = fmt.Sprintf("%v", err)
-			status.LastErrTime = time.Now()
-			return
+		if !status.IsContainer {
```

Comment:
Ditto

Reply:
Yes, for container case, this portion of the code is being ignored.
Would like to revisit to check whether something has to be done in container case.

Reply:
I think what @nordmark is saying -- just insert that same if before the code block and leave the indentation be @spatchineelam 

This is the kind of cosmetics that makes it much easier to review and I think we should take care of it before merge.

Reply:
Taken care of